### PR TITLE
Add stylings

### PR DESCRIPTION
**Pull request by tashisrael**
Added stylings as per the zeplin design

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -13,12 +15,42 @@ function Book({ title, author, id }) {
...
+            <span className="completed">completed</span>
+          </div>
+        </div>
+        <div className="view-progress">
+          <span className="chapter-title">Current chapter</span>
+          <h3 className="chapter-number">Chapter 10</h3>
+          <button type="button" className="btn-2">
+            Update progress
+          </button>
+        </div>
```
Comment (Lucas-Masaba) [inline src/components/Book.js:51]: - [ ] In this project, you are required to parse the design as seen in the zeplin. Please make this card look similar to that in the  design.
### Your Implementation
image
### Required
image

File: src/components/Book.js [hunk 2]
```diff
@@ -13,12 +15,42 @@ function Book({ title, author, id }) {
...
       </div>
     </>
   );
-}
+};
```
Comment (Lucas-Masaba) [inline src/components/Book.js:55]: - [ ] The underline should be grey and should not span from one end of the page to another.
### Your implementation
image

### Required
image

File: src/components/addBook.js [hunk 3]
```diff
@@ -37,14 +38,27 @@ function AddBook() {
...
+        >
+          <option value="" hidden>Category</option>
+          <option value="Action">Action</option>
+          <option value="Fiction">Fiction</option>
+          <option value="Thriller">Thriller</option>
+          <option value="Inspiration">Inspiration</option>
+          <option value="Business">Business</option>
+        </select>
         <button type="submit">Add</button>
       </form>
```
Comment (Lucas-Masaba) [inline src/components/addBook.js:60]: - [ ] Please add similar widths for the input fields and spaces between them as seen in the original zeplin design
### Your Implementation
image
### Required
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @tashisrael  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Design looks similar to that as seen in the zeplin design but needs a few weeks 🏄 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @tashisrael , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**